cmd/serve.go

```diff
@@ -224,13 +224,13 @@ var serveCmd = &cobra.Command{
 			viper.GetBool(postgresMigrationsFlag),
 			viper.GetString(postgresMigrationsSourceFlag),
 			viper.GetBool(hasuraMetadataFlag),
-			viper.GetString(graphqlEndpointFlag),
-			viper.GetString(hasuraMetadataAdminSecretFlag),
+			viper.GetString(hasuraEndpointFlag),
+			viper.GetString(hasuraAdminSecretFlag),
 			logger,
 		)
 
 		metadataStorage := getMetadataStorage(
-			viper.GetString(graphqlEndpointFlag) + "/graphql",
+			viper.GetString(hasuraEndpointFlag),
```

Comment:
you need to add the `+ "/graphql"` you removed

Reply:
true story, thank you.